doc/source/changes/version_0_25_1.rst.inc

```diff
@@ -19,6 +19,11 @@ Fixes
 
 * fixed displaying empty array when starting the viewer or a new session in it.
 
+* fixed default import of `view`, `edit` and `compare` functions in the viewer.
+
+* fixed rezising to content of rows and columns in the viewer
```

Comment:
typo in resizing

Reply:
why not a simpler sentence, eg:
* fixed row and column resizing by double clicking on the edge of an header cell.